src/main/java/org/onecx/search/config/rs/v1/controller/SearchConfigControllerV1.java

```diff
@@ -0,0 +1,124 @@
+package org.onecx.search.config.rs.v1.controller;
```

Comment:
Please check the package structure https://github.com/onecx/onecx-permission-svc/blob/main/src/main/java/org/tkit/onecx/permission/domain/daos/PermissionDAO.java#L1